crates/edr_op/src/chains.rs

```diff
@@ -0,0 +1,80 @@
+use std::sync::LazyLock;
+
+use edr_evm::hardfork::{Activations, ChainConfig, ForkCondition};
+use op_revm::OpSpecId;
+
+/// OP Mainnet chain ID
+pub const OP_MAINNET_CHAIN_ID: u64 = 0xa;
+
+/// OP Mainnet chain config
+///
+/// https://github.com/ethereum-optimism/superchain-registry/blob/51804a33655ddb4feeb0ad88960d9a81acdf6e62/superchain/configs/mainnet/op.toml
+pub static OP_MAINNET_CONFIG: LazyLock<ChainConfig<OpSpecId>> = LazyLock::new(|| ChainConfig {
+    name: "op-mainnet".into(),
+    hardfork_activations: Activations::new(vec![
+        (ForkCondition::Block(105_235_063), OpSpecId::BEDROCK),
+        (ForkCondition::Block(105_235_063), OpSpecId::REGOLITH),
+        (ForkCondition::Timestamp(1_704_992_401), OpSpecId::CANYON),
+        (ForkCondition::Timestamp(1_710_374_401), OpSpecId::ECOTONE),
+        (ForkCondition::Timestamp(1_720_627_201), OpSpecId::FJORD),
+        (ForkCondition::Timestamp(1_726_070_401), OpSpecId::GRANITE),
+        (ForkCondition::Timestamp(1_736_445_601), OpSpecId::HOLOCENE),
+    ]),
+});
+
+/// OP Sepolia chain ID
+pub const OP_SEPOLIA_CHAIN_ID: u64 = 0xaa37dc;
+
+/// OP Sepolia chain config
+///
+/// https://github.com/ethereum-optimism/superchain-registry/blob/51804a33655ddb4feeb0ad88960d9a81acdf6e62/superchain/configs/sepolia/op.toml
+pub static OP_SEPOLIA_CONFIG: LazyLock<ChainConfig<OpSpecId>> = LazyLock::new(|| ChainConfig {
+    name: "op-sepolia".into(),
+    hardfork_activations: Activations::new(vec![
+        (ForkCondition::Block(0), OpSpecId::BEDROCK),
+        (ForkCondition::Block(0), OpSpecId::REGOLITH),
+        (ForkCondition::Timestamp(1_699_981_200), OpSpecId::CANYON),
+        (ForkCondition::Timestamp(1_708_534_800), OpSpecId::ECOTONE),
+        (ForkCondition::Timestamp(1_716_998_400), OpSpecId::FJORD),
+        (ForkCondition::Timestamp(1_723_478_400), OpSpecId::GRANITE),
+        (ForkCondition::Timestamp(1_732_633_200), OpSpecId::HOLOCENE),
+    ]),
+});
+
+/// Base Mainnet chain ID
+pub const BASE_MAINNET_CHAIN_ID: u64 = 8453;
+
+/// Base Mainnet chain config
+///
+/// https://github.com/ethereum-optimism/superchain-registry/blob/51804a33655ddb4feeb0ad88960d9a81acdf6e62/superchain/configs/mainnet/base.toml
+pub static BASE_MAINNET_CONFIG: LazyLock<ChainConfig<OpSpecId>> = LazyLock::new(|| ChainConfig {
+    name: "base-mainnet".into(),
```

Comment:
This seems to be called just "Base" (https://chainlist.org/chain/8453) but I'm happy to follow the pattern of using `*-mainnet` everywhere.

Reply:
I didn't give this much thought initially. "Base" or "base" may be better.

Where will this name show up?

Reply:
It's used in [this error message](https://github.com/NomicFoundation/edr/blob/feat/multichain/crates/edr_evm/src/blockchain/forked.rs#L62), collected at [this point](https://github.com/NomicFoundation/edr/blob/feat/multichain/crates/edr_evm/src/blockchain/forked.rs#L207).

Reply:
If the name is meant for error messages I would go with something like this:
- OP Mainnet
- OP Sepolia
- Base
- Base Sepolia

Existing names use lowercase ("goerli", "kovan", etc.) though, so these could also be "op mainnet", "base sepolia", etc. Let me know if you have a preference.

Reply:
I like your proposal better than the lowercase names. Given that they are in error messages, readability is a priority.

EDR and Hardhat use lowercase for the chain types but those are a different set than the chain names (match Chain ID).